### DISCUSSION TITLE
Auth0 - config.js

### DISCUSSION THREAD

**Discussion by LCLabQU** [l3atbc-datadog/pushkin#95] [General] [answered]
I see two config.js files were created in installing Pushkin-quickstart. Which one has to be edited to match the Auth0 Domain and ClientID ? Doe it matter?
Thanks!

**Comment by conbainbridge** [comment, marked as answer]
For the Auth0 Domain and ClientID you will be editing the file /pushkin/front-end/src/config.js

Just to make sure it setup correctly - where is the second config.js you mention located in your folder structure?

**Reply by LCLabQU** [reply]
Hi Connie, found it (a bit hard to see in the movie). 
I now see a 12 more files with the same name (but I ran a few more steps) - all in subdirectories of node_modules

**Reply by conbainbridge** [reply]
In general, we will almost always want to stay out of any node_modules folders. These are the folders where the software dependencies, and the dependencies for the dependencies all live and get automatically handled when updates and installations occur.

Another place one might find files named "config.js" is in the experiment specific folders - these are for editing the configurations specific to the experiments and will typically have a pretty short list of options to alter, such as what color the font should be.
